<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>iRICMIMainWindow</class>
 <widget class="QMainWindow" name="iRICMIMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>iRIC-MI 0.0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="modelsTab">
       <attribute name="title">
        <string>Models</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="modelsTableView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showGrid">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="addModelButton">
            <property name="text">
             <string>&amp;Add...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addExistingModelButton">
            <property name="text">
             <string>Add &amp;Existing...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeModelButton">
            <property name="text">
             <string>&amp;Remove...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="connectionsTab">
       <attribute name="title">
        <string>Connections</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="connectionsTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addConnectionButton">
            <property name="text">
             <string>&amp;Add...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeConnectionButton">
            <property name="text">
             <string>&amp;Remove...</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="visualizeButton">
            <property name="text">
             <string>&amp;Visualize...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newProjectAction"/>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="closeAction"/>
    <addaction name="separator"/>
    <addaction name="reloadAction"/>
    <addaction name="propertyAction"/>
    <addaction name="separator"/>
    <addaction name="action_Recent_Projects"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newProjectAction"/>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="separator"/>
   <addaction name="reloadAction"/>
  </widget>
  <action name="exitAction">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="newProjectAction">
   <property name="icon">
    <iconset resource="../../guibase/guibase.qrc">
     <normaloff>:/libs/guibase/images/iconNew.svg</normaloff>:/libs/guibase/images/iconNew.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;New Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset resource="../../gui/gui.qrc">
     <normaloff>:/images/iconOpen.svg</normaloff>:/images/iconOpen.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="../../gui/gui.qrc">
     <normaloff>:/images/iconSave.svg</normaloff>:/images/iconSave.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="propertyAction">
   <property name="text">
    <string>&amp;Property...</string>
   </property>
  </action>
  <action name="action_Recent_Projects">
   <property name="text">
    <string>&amp;Recent Projects</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="reloadAction">
   <property name="icon">
    <iconset resource="../../gui/gui.qrc">
     <normaloff>:/images/iconReload.svg</normaloff>:/images/iconReload.svg</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../gui/gui.qrc"/>
  <include location="../../guibase/guibase.qrc"/>
 </resources>
 <connections/>
</ui>
